src/desktop/components/react/stitch_components/StagingBanner.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react"
+import { Banner } from "@artsy/palette"
+
+export const StagingBanner = () => (
+  <Banner backgroundColor="purple100" message="staging.artsy.net" />
```

Comment:
Feels slightly icky to hard-code a domain name that may or not match the location here based on an different config. Can we get this from express somehow instead?

Reply:
`sd.APP_URL`